plugins/utils/comment-nvim.nix

```diff
@@ -29,6 +29,8 @@ in {
         description = "Lines to be ignored while comment/uncomment";
         default = null;
       };
+      preHook = helpers.defaultNullOpts.mkStr "" "Lua function called before (un)comment.";
+      postHook = helpers.defaultNullOpts.mkStr "" "Lua function called after (un)comment.";
```

Comment:
I'd suggest using `helpers.defaultNullOpts.mkNullable types.str "null" "...."` as a type instead, to denote the null value.